doc/source/pmm-admin.rst

```diff
@@ -378,6 +381,49 @@ use a command similar to the following:
 
 For more information, run ``sudo pmm-admin add mysql:metrics --help``.
 
+.. _pmm-admin-add-mongodb-queries:
+
+Adding MongoDB query analytics service
+--------------------------------------
+
+Use the ``mongodb:queries`` alias to enable MongoDB query analytics.
+
+.. note:: Query analytics for MongoDB is experimental.
+   To add it in a non-production environment,
+   use the ``--dev-enable`` option for ``pmm-admin add``.
+
+**USAGE**
+
+.. code-block:: text
+
+   pmm-admin add mysql:queries [NAME] [OPTIONS]
+
+This creates the ``pmm-mongodb-queries-0`` service
```

Comment:
MySQL QAN service used to be with no port. Not sure when that changed. I will update accordingly.